packages/mobile/test/schemas.ts

```diff
@@ -291,6 +291,153 @@ export const v1Schema = {
   },
 }
 
+export const v2Schema = {
```

Comment:
This schema migration should only include changes from v1 schema. Here is an example of a migration: https://github.com/celo-org/celo-monorepo/blob/5c4f07edfb57f5161ff2622b3bdffe0ed7bbb515/packages/mobile/test/schemas.ts#L181